packages/amplify-e2e-tests/src/__tests__/notifications.test.ts

```diff
@@ -1,96 +1,50 @@
-import _ from 'lodash';
 import {
-  addSMSNotification,
```

Comment:
Is this the only e2e test we have for notifications? if so, what concepts do you think should be verified (what's a good e2e test for this PR)

Reply:
this is the only one, it used to test just SMS channel and now was changed to do a push/pull for SMS and In-App Messaging channels. 
not sure what can we do with a e2e test, but we should test that creating resources with older version and then upgrading to the new implementation and updating/adding new channels work as expected. One of the main changes in this PR is that notifications and analytics no longer create 2 distinct pinpoint resources but whoever will share the same resource, also pinpont resource is always managed by the analytics category so adding a notification category will also add a analytics category with the pinpoint resource.

For the above we should test various combinations for when adding/updating analytics/notifications channgels are mixed and matched with old and new implementation

Other things that could be tested are update and delete for the notifications channels (this tests only tests add)